src/igvfd/schemas/changelogs/construct_library.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`construct_library.json`*
 
+### Schema version 2
+
+* Remove `plasmid_map` property. A plasmid map should be submitted to  `documents` property instead.
```

Comment:
in changelogs, we are trying to be very "terse". I think we can drop the "property" from the first sentence. So something like: `Remove "plasmid_map". Use "documents.document_type" instead.`

Reply:
I made the recommended change, saying "Use 'documents' instead." (I left out document_type to indicate that the whole plasmid map document must be attached)